test/web-platform-tests/to-upstream/dom/nodes/ParentNode-querySelector-All-content-dont-upstream.xht

```diff
@@ -0,0 +1,372 @@
+<!DOCTYPE html>
```

Comment:
Can you add a comment explaining why we have a dont-upstream copy of this?

Reply:
Done